src/components/KCheckbox/KCheckbox.vue

```diff
@@ -110,7 +109,7 @@ const emit = defineEmits<{
 const slots = useSlots()
 const attrs = useAttrs()
 
-const inputId = attrs.id ? String(attrs.id) : useUniqueId()
+const inputId = attrs.id ? String(attrs.id) : useId()
```

Comment:
```suggestion
const id = useId()
const inputId = computed(() => attrs.id ? String(attrs.id) : id)
```

I think now this should be a computed ref as we now support dynamic user defined `id`.

Reply:
nit: I'd name differently than `id` just to not get confused with other properties, etc. Something like `defaultId` might be more obvious

Reply:
Fixed